tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/decimaltype/BDecimalValueNegativeTest.java

```diff
@@ -76,5 +76,12 @@ void testDecimalValueNegativeLiteral() {
         BAssertUtil.validateError(negative, i++, "operator '+' not defined for 'float' and 'decimal'", 38, 17);
         BAssertUtil.validateError(negative, i++, "operator '-' not defined for 'float' and 'decimal'", 39, 17);
         BAssertUtil.validateError(negative, i++, "operator '+' not defined for 'decimal' and 'float'", 41, 17);
+        BAssertUtil.validateError(negative, i++, "'9.9999999999999999999999999999999999e6144' is out of range" +
```

Comment:
Shall we change the error msg to not to include the larger number because if overflow number is bigger then error msg also will increase?

Reply:
@gimantha, WDYT?
An error message like `decimal is out of range` is enough?

Reply:
Dont we need to make this consistent with ints and floats?

Reply:
for ints, we give `Integer <int value> too large`

Reply:
In master we dont have `Integer <int value> too large` or `Integer <int value> too small` errors. instead we have `out of range`. I dont think it is good to reformat what user has given originally. Also we cannot reformat to smaller length always right?